plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/core.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import json
+import logging
+from typing import Any, Dict, List, Optional, Union
+
+import nevergrad as ng  # type: ignore
```

Comment:
Please see #476.
i didn't test but I suspect nevergrad has the same issue, please import lazily.
(also show startup timing before and after similar to that in the #476)

Reply:
done